src/embed/dispatch.rs

```diff
@@ -82,7 +82,8 @@ async fn dispatch_loop(rx: Receiver<Request>) {
     }
 }
 
-pub async fn dispatch(db_name: String, rpc: String, data: String) -> Response {
+pub async fn dispatch(db_name: String, rpc: String, data: wasm_bindgen::JsValue) -> Response {
+    let data = data.as_string().unwrap();
```

Comment:
The rpc is private API. If someone uses it directly all bets are off.

Reply:
Agree with fritz. We have adopted a “no panics” policy in repc because we’re embedded in someone else’s memory space. Rust makes it easy and elegant to exhaustively handle error cases. This was just an overlook.

Reply:
Note @phritz killing "the application" here is a somewhat squishy concept. Wasm is a VM - when it dies the app doesn't completely die ... like it doesn't sadtab or segfault. Just the Wasm VM crashes.

In practice I'm sure this makes most apps unusable though and it's not something that we want devs to have to deal with, so this is just a pedantic point.